source/Nevermore/RelationalStore.cs

```diff
@@ -17,14 +17,12 @@ public class RelationalStore : IRelationalStore
     {
         readonly Lazy<RelationalTransactionRegistry> registry;
         readonly Lazy<KeyAllocator> keyAllocator;
-        readonly ITableColumnsCache tableColumnsCache;
```

Comment:
This wasn't ever being used, leftover artifact from when we were spiking how this was going to work